### PR TITLE
[fix](compaction) fix null pointer when retrieving CPU load average

### PR DESCRIPTION
**Pull request by luwei16**
1 When enable_system_metrics = false, system_metrics is a null pointer
2 When BE exits, system_metrics has been destructed, and the compaction background thread may still be running

### CODE REVIEW

**Comment by luwei16** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: be/src/util/system_metrics.cpp [hunk 1]
```diff
@@ -870,7 +870,11 @@ void SystemMetrics::get_disks_io_time(std::map<std::string, int64_t>* map) {
 }
 
 double SystemMetrics::get_load_average_1_min() {
-    return _load_average_metrics->load_average_1_minutes->value();
+    if (!_load_average_metrics) {
```
Comment (gavinchou) [inline be/src/util/system_metrics.cpp:873]: it seems incorrect

**Comment by luwei16** [comment]
run buildall

File: be/src/util/system_metrics.cpp (hunk 1 above)
Comment (luwei16) [reply be/src/util/system_metrics.cpp]: done

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]

**Pull request merged**